### PR TITLE
Bug 1069013 - Make crop and effect buttons aria radio buttons and label ...

### PR DESCRIPTION
**Pull request by eeejay**
...them.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7ad72c45c78f)

File: apps/gallery/index.html [hunk 1]
```diff
@@ -186,25 +186,25 @@ <h1 id="edit-title" data-l10n-id="edit"></h1>
...
 
-        <div id="edit-crop-options" class="edit-options-bar hidden">
-          <a id="edit-crop-none" class="button" data-icon="undo"></a>
-          <a id="edit-crop-aspect-free" class="selected radio button"></a>
-          <a id="edit-crop-aspect-portrait" class="radio button"></a>
-          <a id="edit-crop-aspect-landscape" class="radio button"></a>
-          <a id="edit-crop-aspect-square" class="radio button"></a>
+        <div id="edit-crop-options" class="edit-options-bar hidden" role="radiogroup" data-l10n-id="crop-options">
+          <a id="edit-crop-none" class="button" data-icon="undo" role="button" data-l10n-id="crop-revert"></a>
+          <a id="edit-crop-aspect-free" class="selected radio button" role="radio" aria-checked="true" data-l10n-id="crop-aspect-free"></a>
```
Comment (punamdahiya) [inline apps/gallery/index.html]: We don't need aria-checked="true", as we are setting it in JS.

Comment (eeejay) [inline apps/gallery/index.html]: Same with 'selected' class. I figure it is good to keep them in the same state.

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2e5cb88dbc44)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2d1e819945ec)

**Pull request merged**